ut-stubs/src/cfe_psp_memaccess_api_handlers.c

```diff
@@ -45,7 +45,7 @@ void UT_DefaultHandler_CFE_PSP_MemCpy(void *UserObj, UT_EntryKey_t FuncKey, cons
     /* CFE_PSP_MemCpy(void *dest, const void *src, uint32 size) */
     void *      dest = UT_Hook_GetArgValueByName(Context, "dest", void *);
     const void *src  = UT_Hook_GetArgValueByName(Context, "src", const void *);
-    uint32      size = UT_Hook_GetArgValueByName(Context, "size", uint32);
+    uint32      n    = UT_Hook_GetArgValueByName(Context, "n", uint32);
```

Comment:
Renamed variable was not updated in memcpy() on line 54. I've corrected this in commit e0fd658ebcff4491a203e0a5fb01dd05992aceb0.